docker-jans-monolith/README.md

```diff
@@ -1,17 +1,27 @@
-# Overview
+---
+tags:
+- administration
+- installation
+- quick-start
+- docker compose
+---
 
-**This image is for testing and development purposes only! Use Janssen [helm charts](../charts) for production setups**
+> **Warning**
+> This image is for testing and development purposes only. Use Janssen [helm charts](../charts) for production setups.
 
-Docker monolith image packaging for Janssen.This image packs janssen services including, the auth-server, config-api, fido2, and scim.
+## Overview
 
-## Versions
+Docker monolith image packaging for Janssen. This image packs janssen services including the auth-server, config-api, fido2, and scim.
+
+## Pre-requisites
+
+- [Docker](https://docs.docker.com/install)
+- [Docker compose](https://docs.docker.com/compose/install/)
 
-See [Releases](https://github.com/JanssenProject/docker-jans-monolith/releases) for stable versions. This image should never be used in production.
-For bleeding-edge/unstable version, use `janssenproject/monolith:1.0.4_dev`.
 
 ## Environment Variables
 
-The following environment variables are supported by the container:
+Installation depends on a set of environment variables. These environment variables can be set to customize installation as per the need. If not set, the installer uses default values.
```

Comment:
reword to `Installation depends on the a set of environment variables as shown below. These environment variables can be set to customize the installation. If not set, the installer uses default values.`